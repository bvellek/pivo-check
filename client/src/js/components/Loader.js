import React from 'react';

const Loader = () => (
  <div className="loader-wrapper">
    <div className="loader">Loading...</div>
  </div>
);

export default Loader;
