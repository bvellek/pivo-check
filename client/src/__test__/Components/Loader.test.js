import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Loader from '../../js/components/Loader';

describe('<Loader />', () => {
  it('renders without crashing', () => {
    shallow(<Loader />);
  });
  it('it renders correctly', () => {
    const component = shallow(
      <Loader />
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
