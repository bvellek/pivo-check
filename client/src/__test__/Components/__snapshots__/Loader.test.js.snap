// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> it renders correctly 1`] = `
<div
  className="loader-wrapper"
>
  <div
    className="loader"
  >
    Loading...
  </div>
</div>
`;
